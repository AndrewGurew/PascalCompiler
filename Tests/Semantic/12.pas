var a:double;
begin
a:= 15 and 40.0;
writeln(a);
end