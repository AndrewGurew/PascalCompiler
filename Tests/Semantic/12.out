(3, 9) - operator and is not overloaded for Integer and Double